lms/services/blackboard_api/service.py

```diff
@@ -53,12 +53,34 @@ def list_files(self, course_id):
         :raise ProxyAPIAccessTokenError: if we don't have a Blackboard API
             access token for the current user
         """
-        return self._http_service.get(
+        files = self._http_service.get(
             self._api_url(f"courses/uuid:{course_id}/resources"),
             oauth=True,
             schema=BlackboardListFilesSchema,
         ).validated_data
 
+        return [
+            {
+                "id": file_["id"],
+                "display_name": file_["name"],
+                "updated_at": file_["modified"],
+            }
+            for file_ in files
+        ]
```

Comment:
The `list_files()` method is now doing the work of renaming the `name` and `modified` fields and other excluding fields other than `id`, `display_name` and `updated_at`. This is because the `public_url()` method below now uses the same `BlackboardListFilesSchema` and `public_url()` _doesn't_ want this renaming and excluding, so I moved the renaming and excluding out of `BlackboardListFilesSchema`